config/projects/chefdk.rb

```diff
@@ -65,7 +65,12 @@
   override :rubygems,     version: "2.4.8"
 end
 
+# update to 0.9.1 when that is released
+# master currently has some dependency changes we need
+override :'inspec', version: "master"
+
 override :'test-kitchen', version: "v1.4.2"
+override :'kitchen-inspec', version: "master" # will set when release is tagged
```

Comment:
Thanks @arlimus. We'll probably still pull things in via git (and a tag) for flexibility. Please create a `v0.9.0` tag in https://github.com/chef/kitchen-inspec. `rake release` will do this for you in the future:

```
rake release             # Create tag v0.9.0 and build and push kitchen-inspec-0.9.0.gem to Rubygems
```